src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -985,6 +985,7 @@ private void AddTransactionToWallet(Transaction transaction, TxOut utxo, int? bl
                 var newTransaction = new TransactionData
                 {
                     Amount = amount,
+                    IsCoinBase = transaction.IsCoinBase == false ? (bool?)null : true,
```

Comment:
Wouldn't it be better to use false instead of null?

Reply:
This construct is used for the existing IsCoinStake, I am merely duplicating it.